apps/extension/src/app/ui/utils/README.md

```diff
@@ -1,38 +0,0 @@
-# Stacks UI Utils
```

Comment:
I found this on the way searching for `microStxToStx` and deleting as its obsolete